lib/circlex/struct/wire_instructions.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Circlex.Struct.WireInstructions do
+    import Circlex.Struct.Util
+
+    defstruct [
+      :beneficiary,
+      :beneficiary_bank,
+      :tracking_ref,
+      :virtual_account_enabled
+    ]
+
+    def deserialize(wire_instructions) do
+        %__MODULE__{
+            beneficiary: fetch(wire_instructions, :beneficiary),
+            beneficiary_bank: fetch(wire_instructions, :beneficiaryBank),
+            tracking_ref: fetch(wire_instructions, :trackingRef),
+            virtual_account_enabled: fetch(wire_instructions, :virtualAccountEnabled)
+        }
+    end
+
+    # Accepts Struct.BankAccount, since most wire instructions are hard-coded
+    def serialize(wire_instructions) do
```

Comment:
could improve naming of argument -- if `bank_account` is passed in, we should not call it `wire_instructions`

Reply:
yeah, can you pattern match on the struct to ensure it is what you think it is? As in

```diff
-def serialize(wire_instructions) do
+def serialize(bank_account=%Circlex.Struct.BankAccount{}) do
```

But overall, I'm a little confused by the exact overlap of "BankAccount" and "WireInstructions"?

Reply:
bank account and wire instructions are different objects